janitor/functions/pivot.py

```diff
@@ -83,7 +84,22 @@ def pivot_longer(
         6     setosa   Petal.Width    0.2
         7  virginica   Petal.Width    1.8
 
-        Split the column labels into parts:
+    Convenient, flexible column selection in the `index` via the
+    [`select_columns`][janitor.functions.select.select_columns] syntax:
```

Comment:
Indentation here as well -- indent once to the right.

Same goes for all the diffs that u introduced below.